pkg/tfgen/generate.go

```diff
@@ -88,12 +88,13 @@ type Generator struct {
 type Language string
 
 const (
-	Golang Language = "go"
-	NodeJS Language = "nodejs"
-	Python Language = "python"
-	CSharp Language = "dotnet"
-	Schema Language = "schema"
-	PCL    Language = "pulumi"
+	Golang       Language = "go"
+	NodeJS       Language = "nodejs"
+	Python       Language = "python"
+	CSharp       Language = "dotnet"
+	Schema       Language = "schema"
+	PCL          Language = "pulumi"
+	RegistryDocs Language = "registry-docs"
```

Comment:
We've talked a lot about why it makes sense for `RegistryDocs` to be a "language". Can you add a comment here summarizing why this makes sense?